plugins/kubernetes-node/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@backstage/plugin-kubernetes-node",
+  "description": "Node.js library for the kubernetes plugin",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "node-library"
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "devDependencies": {
+    "@backstage/cli": "workspace:^"
+  },
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@backstage/backend-plugin-api": "workspace:^",
+    "@backstage/plugin-kubernetes-backend": "workspace:^"
```

Comment:
don't think this should be here right?

Reply:
This becomes a cyclical dependency, would need to move all the things that its depended on from here to the node package itself and then re-export them again from `@backstage/plugin-kubernetes-backend` in order to not break that API if that's what you wanted?